Dockerfile.service-controller

```diff
@@ -11,15 +11,13 @@ RUN make
 FROM node:12.18 AS console-builder
 
 WORKDIR /skupper-console/
-ADD https://github.com/skupperproject/gilligan/archive/master.tar.gz .
-RUN tar -zxf master.tar.gz
-WORKDIR ./gilligan-master
-RUN yarn install && yarn build
+ADD https://github.com/skupperproject/gilligan/releases/download/1.0.0/console.tgz
```

Comment:
missing ' .' at the end

Reply:
True, I guess I removed it by mistake when I converted from my fork to skupperproject.
I will fix.